Microsoft/WebTemplateStudio
# Angular all folders name convention

Github Issue: #668, #681
This change is to address one of the suggestions made by angular exeperienced users and experts: 
1. "all folders and symbol names should be in **lower kebab case**, eg, Blank --> blank, Master_Details.module.ts --> master-details.moduke.ts" 
2. Output() properties should not be suffixed with EventEmitter

closes #668

**How to test:**
1. generate an angular project with all 4 type pages selected
2. open the project in vs code and check the folder names 
* there is an exception for master_detail folder name and master_detail.module.ts file, these two can not be addressed at the moment cause it has to do with CoreTemplateStudio.

**Expected Result:** 
![name-convention](https://user-images.githubusercontent.com/12874317/59062439-ac42bd80-885a-11e9-8767-90e3ce042bc9.PNG)
